src/digital_identity_service/share.qr.code.create.result.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+const Validation = require('../yoti_common/validation');
+
+/**
+ * The share QR code result
+ *
+ * @class ShareQrCodeResult
+ */
+module.exports = class ShareQrCodeResult {
```

Comment:
Shouldn't this be ShareQrCodeCreateResult ?

Reply:
👍